### PR TITLE
Change PHPDoc according to WP-CLI Standard Command

### PR DESCRIPTION
**Pull request by mehrshaddarzi**

### CODE REVIEW

File: command.php [hunk 1]
```diff
@@ -15,64 +15,78 @@
 
 /**
  * WP-CLI PO command for satellitewp/po
+ *
+ *  ## EXAMPLES
+ *
+ *      # Merge for basic usage.
```
Comment (maximejobin) [inline command.php]: I'd suggest: Merge for the the URL of a plugin or theme found on the WordPress repository

File: command.php [hunk 2]
```diff
@@ -15,64 +15,78 @@
...
+ *      $ wp po merge fr-ca fr https://wordpress.org/plugins/wordpress-seo/
+ *
+ *      # Merge for Custom locale.
```
Comment (maximejobin) [inline command.php]: I'd suggest: Merge for the translation URL of a project in the WordPress repository (including meta and apps)

File: command.php [hunk 3]
```diff
@@ -15,64 +15,78 @@
...
+ *      $ wp po merge fr-ca fr https://translate.wordpress.org/locale/fr-ca/default/wp-plugins/wordpress-seo/
+ *
+ *      # Merge for Custom version of WordPress.
```
Comment (maximejobin) [inline command.php]: I'd suggest: Merge for the core version of WordPress

**Review by maximejobin — CHANGES_REQUESTED** [review]
Great work! Thank you for this PR. I've suggested some changes for uniformity. Then, I'll approve your PR.

**Comment by mehrshaddarzi** [comment]
@maximejobin 

Done.
please check
https://github.com/SatelliteWP/po-merger/pull/9/commits/65caa5986503ce648f22cbf9173c4f1310607271

**Pull request merged**